server/src/main/scala/io/delta/standalone/internal/DeltaCDFErrors.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.standalone.internal
+
+object DeltaCDFErrors {
+  def multipleCDFBoundary(position: String): Throwable = {
+    new IllegalArgumentException(s"Multiple $position arguments provided for CDF read. Please " +
+      s"provide one of either ${position}Timestamp or ${position}Version."
+    )
+  }
+
+  def noStartVersionForCDF: Throwable = {
```

Comment:
Should we call it noStartParamForCDF?

Reply:
I more prefer to keep it the same as DeltaErrors in our code base.

Reply:
sg.